### PR TITLE
fix: handling of app scheme for nfk and atb

### PR DESCRIPTION
**Pull request by mikaelbr**
Fixes https://github.com/AtB-AS/kundevendt/issues/1920

Came across this issue when testing for https://github.com/AtB-AS/kundevendt/issues/1776 and https://github.com/AtB-AS/kundevendt/issues/2114.

Sets up:

### AtB
```
# Dev
atb-dev://

# Staging
atb-staging://

# Production
atb://
```

### NFK

```
# Dev
nfk-dev://

# Staging
nfk-staging://

# Production
nfk://
```

Based on `APP_SCHEME` in env-folders. This allow us to override based on env and org and easily add support for other orgs.

[See proposed docs](https://github.com/AtB-AS/mittatb-app/blob/b5c73ddddbce5e2c879f33d641ef84444d29b443/docs/OrgConfig.md)

---

Tested on:

- [x] iOS
- [x] Android

This can be a big issue if anyone has both Reis and AtB app running.

---

## How to test locally:

Make sure you have the latest changes and run setup: E.g. `yarn setup dev atb`

```
npx uri-scheme open 'atb-dev://ticketing?transaction_id=321321&payment_id=321321' --android
```

```
npx uri-scheme open 'atb-dev://ticketing?transaction_id=321321&payment_id=321321' --ios
```

## How to test in staging

Completing a ticket purchase should redirect back to active tickets. Using a proxy it should say something like `atb-staging://...`

### CODE REVIEW

**Review by gorandalum — APPROVED** [review]
I think this looks good! :+1:

While you are at it and if you have time, could it also be possible to add some value specifying the environment to the linking and the payment redirect url prefix? So in staging it would be `atb-staging` or something like it.

If buying ticket with the staging app now it redirects back to the store app, and you have to switch app back to the staging app, and there manually cancel the terminal and navigate to the tickets screen.

**Comment by mikaelbr** [comment]
I'll look into it. Probably means introducing a new env var instead of using orgid directly.

**Comment by mikaelbr** [comment]
I propose doing that change as a separate solution and to get this included in 1.23, @gorandalum . I can test this and mark it as ready for review when that is done.

**Comment by mikaelbr** [comment]
I updated to support multiple envs as it was just as easy when fixing this.

**Review by rosvik — APPROVED** [review]
LGTM 👍

File: docs/OrgConfig.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+```
+npx uri-scheme open 'atb-staging://ticketing?transaction_id=321321&payment_id=321321' --android
+```
+
+```
+npx uri-scheme open 'atb-staging://ticketing?transaction_id=321321&payment_id=321321' --ios
+```
+
+This should open "Active tickets" tab in the app.
```
Comment (gorandalum) [inline docs/OrgConfig.md:29]: Great!

**Review by gorandalum — APPROVED** [review]

**Review by toretefre — APPROVED** [review]

**Pull request merged**